spec/models/comment_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  user = User.create(
+    Name: 'Tom',
+    Photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+    Bio: 'Teacher from Mexico.',
+    PostsCounter: 0
+  )
+
+  post = Post.create(
+    Title: 'my title',
+    Text: 'Hello World!',
+    CommentsCounter: 0,
+    LikesCounter: 0,
+    user_id: user.id
+  )
+
+  Comment.create(
+    user_id: user.id,
+    post_id: post.id,
+    Text: 'A nice post'
+  )
```

Comment:
- [ ] Your specs are not passing, and in this case, the error indicates an unknown attribute `post_id` for Comment
![image](https://user-images.githubusercontent.com/46839548/213888175-da84ee6b-f69b-414d-9f36-8b8036690276.png)
I dug around a bit and realized that in your Schema, there is no `post_id` attribute or column for the `Comment` model. I think that is a good place to start in order to solve this problem 👍